index.html

```diff
@@ -575,29 +575,29 @@ <h1>Introduction</h1>
     {
         "@context": [
             "https://www.w3.org/2019/wot/td/v1",
-            { "saref": "https://w3id.org/saref#" }
+            { "eg": "http://example.org/iot#" }
```

Comment:
Example 27 uses "iot" as prefix. However, example 21 uses "eg". We should use always the same prefix for  "http://www.example.org/iot" for any example in the spec.

Reply:
In the latest committed version, I don't see the `iot` prefix anymore?